docs/usecases.rst

```diff
@@ -20,6 +20,17 @@ Use-cases for libCellML
       a. one component encapsulating two children
       #. one component encapsulating a single child which in turn encapsulates a single child
       #. an invalid cyclical encapsulation hierarchy
+      
+   #. manipulation of a model with multi-level component encapsulation hierarchy
+   
+      a. remove a component
+      #. change the name of a top-level component
+      #. change the name of an encapsulated child component
+      #. replace one component with a new component
+      #. take a component
```

Comment:
'take' is ambiguous to me, could mean either get a component from a model (model is unchanged) or remove a component from a model.